UM-USElab/gradecraft-development
# Perform enqueued ahoy sidekiq job GeocodeV2

### Status
**READY**

### Description
adds ahoy jobs to sidekiq config so enqueued jobs of type ahoy are executed

This job (https://www.rubydoc.info/gems/ahoy_matey/Ahoy/GeocodeV2Job#) was being enqueued with each new visit but never performed. 

Ahoy by default creates a new queue of jobs called "ahoy", which was never configured to execute in the priority of sidekiq jobs.

### Related PRs
#4415 - Ahoy! 

### Steps to Test or Reproduce
remove this line and visit the browser on any GC environment, create a new visit (or multiple by using `ahoy.reset();` in browser console. See that new GeoCodeV2 jobs are being enqueued but never performed. 


### Impacted Areas in Application
* Ahoy visits

### Future notes 
EB: From looking at the Ahoy documentation https://github.com/ankane/ahoy and this file: https://github.com/ankane/ahoy/blob/master/app/jobs/ahoy/geocode_v2_job.rb it seems like this job should be gathering location data on the user and storing it in the ahoy visit, at this time it does not seem like that information is being stored.
